sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/implementation/models/Recurrence.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.feature.management.implementation.models;
+
+/**
+ * A recurrence definition describing how time window recurs
+ * */
+public class Recurrence {
```

Comment:
We should add the Annotation to all of the models to ignore unknown fields. It will save us at some point if a field is added to the portal by default. Wouldn't be the first time.

Reply:
Updated